tauri/src/components/ui/participant-row-wo-livekit.tsx

```
@@ -147,11 +147,12 @@ export const ParticipantRow = (props: { user: components["schemas"]["BaseUser"]
           sounds.ringing.stop();
           sounds.callAccepted.play();
           tauriUtils.showWindow("main");
+          const settings = await tauriUtils.getUserSettings();
```

Biome (2.4.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)